### DISCUSSION TITLE
Fluxstore Listing - featured listings to appear firstly

### DISCUSSION THREAD

**Discussion by merorose** [inspireui/support#5777] [New Feature/Ideas Requests]
Basically in listings websites, users pay more to show their listings at first of search results,
but app doesn't do that, while website (Listeo) already did
need to apply featured listings to show first at top of search results

**Comment by ElsaidMahmous** [comment]
fluxstore listing needs some work. and a major update, with new features, namely the booking functions provided by themes, for example Listeo
